src/lsd/importers/fits.py

```diff
@@ -3,122 +3,130 @@
-                hdus.append(hdu_name)
+            if hdu_name[0] == '#':
+                hdu_name = int(hdu_name[1:])
+            hdus.append(hdu_name)
 
-	# Create a FITS importer
-	ldr = FITSImporter(db, args.table, usecols, dtype, setcols=setcols, hdus=hdus)
+    # Create a FITS importer
+    ldr = FITSImporter(db, args.table, usecols, dtype, setcols=setcols, hdus=hdus)
 
-	# Return importer and a list of chunks
-	return ldr, args.file
+    # Return importer and a list of chunks
+    return ldr, args.file
 
 def csv_list(value):
-	return [ (s.strip()) for s in value.split(',') ]
+    return [ (s.strip()) for s in value.split(',') ]
 
 def add_arg_parsers(subparsers):
-	"""
-		This function is part of the importer API.
-		
-		It must add the required parser(s) and set the get_importer callback.
-	"""
-	parser = subparsers.add_parser('fits', help=__doc__)
-	parser.set_defaults(get_importer=get_importer)
-	parser.add_argument('table', help='Name of the table into which to import', type=str)
-	parser.add_argument('file', help='One or more fits files. If ending in .gz or .bz2, they are assumed to be compressed.', type=str, nargs='+')
-	parser.add_argument('-c', '--cols', help='Comma separated list of <colname>:<file_column>, where <file_column> is the name of the column in the FITS file that is to be loaded into <colname>.', type=str)
-	parser.add_argument('--import-primary-key', help='The input files contain the primary key values. Load these instead of automatically asigning new ones.', default=False, action='store_true')
-	parser.add_argument('--set', help='Comma separated list of <colname>=<value>, giving the values to which the specified columns are to be set. The columns given here must not appear in argument to --cols.', default=[], type=csv_list)
-	parser.add_argument('--hdus', help="Comma separated list of HDU names (or HDU indices) to import. The values are interpreted as a HDU names, unless they start with a '#', in which case they're interpreted as an index (e.g., #1 refers to the first HDU after the primary one).", default=['#1'], type=csv_list)
+    """
+        This function is part of the importer API.
+        
+        It must add the required parser(s) and set the get_importer callback.
+    """
+    parser = subparsers.add_parser('fits', help=__doc__)
+    parser.set_defaults(get_importer=get_importer)
+    parser.add_argument('table', help='Name of the table into which to import', type=str)
+    parser.add_argument('file', help='One or more fits files. If ending in .gz or .bz2, they are assumed to be compressed.', type=str, nargs='+')
+    parser.add_argument('-c', '--cols', help='Comma separated list of <colname>:<file_column>, where <file_column> is the name of the column in the FITS file that is to be loaded into <colname>.', type=str)
+	parser.add_argument('--cols-file', help='A file with two, whitespace delimited columns, <colname> <file_column>, having the same meaning as arguments to --cols', type=str)
```

Comment:
This is taken from the text importer as well. I think this line contains a tab, which will have to be converted to spaces.